packages/lib/src/io-util.ts

```diff
@@ -74,3 +74,31 @@ export function stdinIsTTY(): boolean {
 export function stdoutIsTTY(): boolean {
 	return process.stdout.isTTY
 }
+
+/**
+ * Check for existence of YAML file.
+ * Enforces official file extension and warns user if incorrect file detected.
+ */
+export function yamlExists(filepath: string): boolean {
+	const parsedPath = path.parse(filepath)
+
+	if (parsedPath.ext !== '.yaml') {
+		return false
```

Comment:
Not a big deal but I think I would throw an exception here since this method isn't really valid for a non-yaml file (this would makes `false` truly mean only the file does not exist).

Reply:
Good call.